sklearn_numba_dpex/kmeans/kernels/compute_labels.py

```diff
@@ -20,17 +20,17 @@ def make_label_assignment_fixed_window_kernel(
     n_samples, n_features, n_clusters, sub_group_size, work_group_size, dtype, device
 ):
     window_n_centroids = sub_group_size
-    centroids_window_width = window_n_centroids + 1
```

Comment:
Adding +1 here was random try of a trick around bank conflict. But after reading some more on the subject of bank conflicts, I think it makes no sense in this kernel. It doesn't show any performance difference on benchmarks either. Let's just remove it, it also overcomplicated the prediction on local memory usage.